src/act.drive.cpp

```diff
@@ -955,7 +960,7 @@ ACMD(do_control)
 ACMD(do_subscribe)
 {
   struct veh_data *veh = NULL;
-  struct obj_data *obj;
+  struct obj_data *obj, *cyber;
```

Comment:
If you're not referencing this value anywhere other than the for loop, you can just initialize it there.
`for (struct obj_data *cyber = ch->cyberware;...`

Reply:
That's pretty useful info going forward